src/test/remoteconfig/rollback.spec.ts

```diff
@@ -34,90 +29,56 @@ const latestTemplate: RemoteConfigTemplate = createTemplate("115", "2020-08-06T2
 const rollbackTemplate: RemoteConfigTemplate = createTemplate("114", "2020-08-07T23:11:41.629Z");
 
 describe("RemoteConfig Rollback", () => {
-  let sandbox: sinon.SinonSandbox;
-  let apiRequestStub: sinon.SinonStub;
-
-  beforeEach(() => {
-    sandbox = sinon.createSandbox();
-    apiRequestStub = sandbox.stub(api, "request").throws("Unexpected API request call");
-  });
-
   afterEach(() => {
-    sandbox.restore();
+    expect(nock.isDone()).to.equal(true, "all nock stubs should have been called");
+    nock.cleanAll();
   });
 
   describe("rollbackCurrentVersion", () => {
     it("should return a rollback to the version number specified", async () => {
-      apiRequestStub.onFirstCall().resolves({ body: latestTemplate });
+      nock(remoteConfigApiOrigin)
+        .post(`/v1/projects/${PROJECT_ID}/remoteConfig:rollback?versionNumber=${115}`)
+        .reply(200, latestTemplate);
 
       const RCtemplate = await remoteconfig.rollbackTemplate(PROJECT_ID, 115);
 
       expect(RCtemplate).to.deep.equal(latestTemplate);
-      expect(apiRequestStub).to.be.calledOnceWith(
-        "POST",
-        `/v1/projects/${PROJECT_ID}/remoteConfig:rollback?versionNumber=` + 115,
-        {
-          auth: true,
-          origin: api.remoteConfigApiOrigin,
-          timeout: 30000,
-        }
-      );
     });
 
-    it("should reject invalid rollback version number", async () => {
-      apiRequestStub.onFirstCall().resolves({ body: latestTemplate });
+    // TODO: there is no logic that this is testing. Is that intentional?
```

Comment:
These are just going to sit around unless someone explicitly looks at them, can you loop in an appropriate person from RC to comment?